packages/govern/internal/actions/lib/AbstractAction.ts

```diff
@@ -67,5 +80,6 @@ export default abstract class AbstractAction {
    */
   public execute(): Promise<any> {
```

Comment:
Execute can get defined as ``async`` which in the end would result in cleaner code.